src/client/components/List/index.js

```diff
@@ -10,11 +11,15 @@ export type Props = {
   pageSize: number,
   total: number,
   activePage: number,
-  children?: number | string | React.Element <*> | Array < any >,
-  onPageSelect?: Function,
+  children?: number | string | React.Element<*> | Array < any >,
+  onPageSelect ?: Function,
```

Comment:
so the problem in the line above `number | string | React.Element<*> | Array < any >`, probably eslint doesn't know how to recognize this flow construction